src/core_init_atmosphere/Registry.xml

```diff
@@ -137,6 +137,11 @@
                      description="The number of months in a year"
                      possible_values="Positive integer values"/>
 
+	        <nml_option name="config_nsoilcat"              type="integer"       default_value="16"
```

Comment:
It doesn't look like `config_nsoilcat` is used anywhere, so it seems like we could remove it from this PR branch.

Reply:
@mgduda Mike, this variable will be used in the following PRs that will compute the fractional content of soil types in each grid cell. And the 3-d array of fractional soil types will be dimensioned as nsoilcat.

Reply:
@tanyasmirnova Apologies for the long delay in following up on this PR. If the `config_nsoilcat` option will be used in a future PR, perhaps we could add it to the `Registry.xml` file at that time?

Reply:
@mgduda Michael, yes, we can add config_nsoilcat later.